integration/spark2/src/main/scala/org/apache/carbondata/spark/rdd/CarbonDataRDDFactory.scala

```diff
@@ -588,6 +591,21 @@ object CarbonDataRDDFactory {
     }
   }
 
+  /**
+   * clear datamap files for segment
+   */
+  private def clearDataMapFiles(carbonTable: CarbonTable, segmentId: String): Unit = {
+    try {
+      DataMapStoreManager.getInstance().getAllDataMap(carbonTable).asScala
+        .filter(_.getDataMapSchema.isIndexDataMap).foreach { p =>
+        p.deleteDatamapData(List(new Segment(segmentId)).asJava)
```

Comment:
move `List(new Segment(segmentId)).asJava` before the loop and move `foreach` to next line, to make it better formatted

Reply:
OK